headsplugin-api/src/main/java/com/github/cc007/headsplugin/api/business/services/heads/CategoryUpdater.java

```diff
@@ -0,0 +1,32 @@
+package com.github.cc007.headsplugin.api.business.services.heads;
+
+import java.util.Collection;
+
+public interface CategoryUpdater {
```

Comment:
This class should not be public and is removed in the rewrite

Reply:
This version of the api is only based on the spring version. Aside from that, the api should definitely provide a way to manually trigger an update of the categories, because they might have updated during the uptime of the server and you don't want the server having to restart to update and you also don't want to update everytime you want to access a category or needlessly update on a timed schedule. Therefore a dedicated category updater is needed